### PR TITLE
Add FastQC support. Remove multiqc_config.yaml file from conf folder.

### PR DESCRIPTION
**Pull request by santiagorevale**
Many thanks to contributing to nf-core/scrnaseq!

Please fill in the appropriate checklist below (delete whatever is not relevant). These are the most common things requested on pull requests (PRs).

## PR checklist
 - [x] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] If necessary, also make a PR on the [nf-core/scrnaseq branch on the nf-core/test-datasets repo]( https://github.com/nf-core/test-datasets/pull/new/nf-core/scrnaseq)
 - [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [ ] Make sure your code lints (`nf-core lint .`).
 - [x] Documentation in `docs` is updated
 - [x] `CHANGELOG.md` is updated
 - [ ] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/scrnaseq/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,27 +1,35 @@
...
-* Salmon to 1.1.0
-* Samtools to 1.10
-* GFFRead to 0.11.7
-* Kallisto to 0.46.2
-* BUSTools to 0.40.0
+* Added FastQC `0.11.9`
+* Update Salmon `1.0.0` -> `1.1.0`
+* Update Samtools `1.9` -> `1.10`
+* Update GFFRead `0.11.6` -> `0.11.7`
+* Update Kallisto `0.46.0` -> `0.46.2`
```
Comment (apeltzer) [inline CHANGELOG.md]: @winni2k commented these days in one of the issues, that there is a problem with the latest kallisto version - maybe we stick to the 0.46.1 instead? https://github.com/nf-core/scrnaseq/issues/21#issuecomment-586206252

Comment (santiagorevale) [reply CHANGELOG.md]: Oh, sorry, it wasn't me who bumped kallisto. That change is already implemented in dev. I just re-formatted the line in the `CHANGELOG.md` file. What do you want me to do, then? Should I fix it and change it back to 0.46.0? Or maybe to 0.46.1 like winni2k suggested?

Comment (apeltzer) [reply CHANGELOG.md]: I think we should go back to 0.46.1 then - if you can do that in your PR, that would be fab! :)

Comment (santiagorevale) [reply CHANGELOG.md]: Done. I've already pushed the changes. I guess they should be reflected here eventually?

Comment (apeltzer) [reply CHANGELOG.md]: They automatically show up here immediately after your push :-)

**Comment by apeltzer** [comment]
Some small errors from the Dockerfile:

```

ERROR: Test Failures:
  http://nf-co.re/errors#9 : Could not find Dockerfile file string: RUN conda env export --name nf-core-scrnaseq-1.0.1dev > nf-core-scrnaseq-1.0.1dev.yml
  http://nf-co.re/errors#9 : Could not find Dockerfile file string: ENV PATH /opt/conda/envs/nf-core-scrnaseq-1.0.1dev/bin:$PATH
```

Could you add these in too?

**Comment by santiagorevale** [comment]
> Some small errors from the Dockerfile:
> 
> ```
> 
> ERROR: Test Failures:
>   http://nf-co.re/errors#9 : Could not find Dockerfile file string: RUN conda env export --name nf-core-scrnaseq-1.0.1dev > nf-core-scrnaseq-1.0.1dev.yml
>   http://nf-co.re/errors#9 : Could not find Dockerfile file string: ENV PATH /opt/conda/envs/nf-core-scrnaseq-1.0.1dev/bin:$PATH
> ```
> 
> Could you add these in too?

Done. Sorry for the delay.

**Comment by santiagorevale** [comment]
Hey @apeltzer, do you know what's going on with the tests? They seem to hang for a long time and then end up being cancelled.

**Comment by grst** [comment]
Hi @santiagorevale, 

I'm afraid that after the DSL2 conversion, this PR is so outdated that it basically needs to be rewritten from scratch. 
However, we would still be interested in having a fastqc module as part of the pipeline. 

Do you want to give it a go? 

Cheers, 
Gregor

**Pull request closed without merging**